<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>493</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Moony Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="moony_resource.qrc">
    <normaloff>:/images/images/moon.gif</normaloff>:/images/images/moon.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="picLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="moony_resource.qrc">:/images/images/moonypic.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="fsfeaturesgroupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>306</width>
            <height>344</height>
           </size>
          </property>
          <property name="title">
           <string>File System Features:</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>30</y>
             <width>285</width>
             <height>182</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="fsLabel">
                <property name="text">
                 <string>File System Selected:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="fstypeLabel">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>vfat</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="feat1Label">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Label</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="feat2Label">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Volume-ID</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="feat3Label">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>FAT Size</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="feat4Label">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Number of Fats</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="feat5Label">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Disk sectors per cluster</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLineEdit" name="feat1LineEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="feat2LineEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="feat3LineEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="feat4LineEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="feat5LineEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="filesystemLabel">
          <property name="text">
           <string>File System</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>vfat</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ext2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ext3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>xfs</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Device File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>/dev/sdb</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/dev/sdb1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/dev/sdb2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/dev/sda</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/dev/sda1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/dev/sda2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionStart_Formatting"/>
    <addaction name="actionStop_Formatting"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_Moony_Creator"/>
    <addaction name="action_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart_Formatting">
   <property name="icon">
    <iconset resource="moony_resource.qrc">
     <normaloff>:/images/images/start.png</normaloff>:/images/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Start &amp;Formatting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionStop_Formatting">
   <property name="icon">
    <iconset resource="moony_resource.qrc">
     <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop Formatting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="moony_resource.qrc">
     <normaloff>:/images/images/quit.png</normaloff>:/images/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About_Moony_Creator">
   <property name="icon">
    <iconset resource="moony_resource.qrc">
     <normaloff>:/images/images/moon.gif</normaloff>:/images/images/moon.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;About Moony Creator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+?</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="moony_resource.qrc">
     <normaloff>:/images/images/help.png</normaloff>:/images/images/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="moony_resource.qrc"/>
 </resources>
 <connections/>
</ui>
